traveler-ios-ui/TravelerKitUI/ViewControllers/OrderDetailViewController.swift

```diff
@@ -93,8 +93,12 @@ open class OrderDetailViewController: UITableViewController {
         purchasedProduct = order!.products[indexPath.row]
 
         tableView.deselectRow(at: indexPath, animated: true)
-        
-        performSegue(withIdentifier: "productDetailSegue", sender: nil)
+
+        if purchasedProduct is PurchasedPartnerOfferingProduct  {
```

Comment:
To future-proof, should probably do something more along the lines of:
`switch purchasedProduct?.purchaseType
        {
        case .booking, .parking:
            performSegue(withIdentifier: "productDetailSegue", sender: nil)
        default:
            Log("Details not available for purchase type.")
        }`
The same needs to be applied to ItineraryResultViewController